proto/gen/openapi/openapi.yaml

```diff
@@ -3933,7 +3938,7 @@ paths:
           description: An unexpected error response.
       summary: List Topics
       tags:
-        - Topics (v1alpha2)
+        - Topics
     post:
       description: Create a [topic](https://docs.redpanda.com/current/deploy/deployment-option/cloud/create-topic/).
```

Comment:
This doc link should also be changed to https://docs.redpanda.com/redpanda-cloud/get-started/create-topic/